keyboards/cipulot/ec_prox/ansi_iso/ec_switch_matrix.c

```diff
@@ -0,0 +1,183 @@
+/* Copyright 2023 Cipulot
```

Comment:
if possible, this should be moved to something like `keyboards/cipulot/lib` or `keyboards/cipulot/common`, and used by the different boards that need it.

Reply:
Hi there @drashna , that is the plan eventually. As of now the file in question changes depending  on the board revision and project all together. The functions are gonna be made "board agnostic" in a future revision.
As of now tho it is better to leave it in the keyboard specific folder.